charts/nirmata/README.md

```diff
@@ -81,7 +81,7 @@ kubectl -n kyverno create configmap <e.g. ca-store-cm> --from-file=custom-ca.pem
 **Install the Kyverno chart:**
 
 ```console
-helm install kyverno --namespace kyverno --create-namespace nirmata/kyverno
+helm install kyverno --namespace kyverno --create-namespace nirmata/kyverno --set licenseManager.licenseKey=<license key obtained from Nirmata support or SAAS>,licenseManager.apiKey=<api key obtained from Nirmata support or SAAS>
```

Comment:
The prerequisite section `Get token from Nirmata support` should be replaced with a section to get the license key and apikey from support@nirmata.com

Reply:
Changed